crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -213,11 +213,14 @@ pub fn extract_core_3d_camera_phases(
 ) {
     for (entity, camera) in &cameras_3d {
         if camera.is_active {
-            commands.get_or_spawn(entity).insert_bundle((
-                RenderPhase::<Opaque3d>::default(),
-                RenderPhase::<AlphaMask3d>::default(),
-                RenderPhase::<Transparent3d>::default(),
-            ));
+            commands
+                .get_or_spawn(entity)
+                .expect("Entity ID is already reserved.")
```

Comment:
Should all these internal uses include the entity id that is panics on?

```suggestion
                .unwrap_or_else(|| panic!("Entity ID: {entity} is already reserved."))
```

Reply:
It should probably include both id's. The existing and the one you tried to spawn, but that probably means that get_or_spawn would need to return a result instead.

Reply:
Hmm, actually it panics if the index of both entities coincide and generations are differing. So you want to report the generation as well. That's interesting, but I think we can do it in a future PR. I think fixing the bug is higher priority here.

I think though that the message can be changed to display the entity for now.